src/pages/apis/drug/remsspl/_explorers.yaml

```diff
@@ -0,0 +1,38 @@
+effectiveTime:
```

Comment:
@violetcrestedwren A number of sample queries currently return no matches: REMS SPL record with a medication guide, REMS SPL records by Set ID, Search REMS SPL records by Brand Name. We'll need to update these

Reply:
fixed these